problem-20.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+/* Declares two arrays of float type of size 10, takes input from user in both arrays
+and determines if there is any element in array 1 which has smaller value than any
+ of the element in array 2 */
+
+int main()
+{
+    float array_1[10], array_2[10];
+    int flag = 0;

```

Comment:
why are you named this as `flag`?

Reply:
It basically stands for number of elements that are smaller in array 1 than any of the elements in array 2 but changed it now